### ISSUE TITLE
Up/down arrow keys should cycle through menu options

### ISSUE THREAD

**Issue by mazameli** [metabase/metabase#5686]
When using these kinds of menus, where I'm filtering the list (and therefore my fingers are on the keyboard), it's annoying that I can't use the up/down arrow keys to highlight a selection and hit enter to select it.

![screen shot 2017-08-04 at 10 04 28 am](https://user-images.githubusercontent.com/2223916/28978821-71764a42-78fc-11e7-8308-a307d3af330f.png)

![screen shot 2017-08-04 at 10 04 35 am](https://user-images.githubusercontent.com/2223916/28978820-6f10089c-78fc-11e7-907d-21c19ae5d6e0.png)

⬇️  **Please click the 👍 reaction instead of leaving a `+1` or 👍  comment**

**Comment by mazameli** [comment]
@attekei, now that you've implemented the arrow keys cycling through options for the New Question Flow, how hard do you think it would be to similarly implement it for other popover lists like these?

**Comment by attekei** [comment]
@mazameli Should be pretty easy to achieve by listening to key presses in `AccordianList` component. It would block the default up/down key behavior which is scrolling the list up and down. Not sure should the scroll position be updated as well if you click down so many times that highlighted search result is below the visible area of popover.

**Comment by mazameli** [comment]
I think my expectation would be that if you press the down arrow key enough to go down past the last currently visible item, then yes, it would scroll the list down to the next item and highlight it.

**Comment by camsaul** [comment]
@mazameli isn't this the same thing you opened in #2369?

Don't forget that you can use the search bar to see if an issue is already open. :wink:

**Comment by dannyeuu** [comment]
@flamber  now with new features on global search and new SQL question search I miss this feature to use with search results, BTW my brain try to do instantly an arrow down and as it is now a standard feature =)

**Comment by flamber** [comment]
@dannyeuu Requires that we update the components, which is something we're working on, but a long process - #15551

**Comment by mazameli** [comment]
This has worked for a while now thanks to new Mantine-based. components.
